examples/closures/closures.bal

```diff
@@ -52,7 +52,7 @@ public function main() {
     int result1 = foo(3);
     io:println("Answer: " + result1);
 
-    // This function invocation shows multiple levels of lambda functions
+    // This function invocation shows multiple levels of anonymous functions
```

Comment:
Shall we change as follows:
"// Function invocation that represents multiple levels of anonymous functions"